src/vueWrapper.ts

```diff
@@ -57,22 +62,26 @@ export class VueWrapper<T extends ComponentPublicInstance> {
   classes(): string[]
   classes(className: string): boolean
   classes(className?: string): string[] | boolean {
-    return new DOMWrapper(this.element).classes(className)
+    return new DOMWrapper(this.element).classes(className!)
```

Comment:
Yes indeed, I'm not sure why it isn't accepting it given as you said the signature is the same.
At the same this this will not be relevant when #375 gets merged.
Regardless in the interim I'll update it to the conditional.